.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Unit Tests and Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Do we not have a similar per-release branch here too?

Reply:
I am not sure what other branches we should include (we definitely exclude the private/* branches but shall we include the platform9/* branches? There are only two). 
 
IMO, let's get started with master and then we can add branches later.